oauth2/provider/scope-authorize.php

```diff
@@ -76,6 +81,23 @@
                     </div>
                 </div>
             </div>
+            <?php if (true == $offline_requested) : ?>
+                <div class="row mt-3">
+                    <div class="col-12">
+                        <div class="alert alert-warning">
+                            <p>
+                                <?php echo xlt("This application has requested offline access to your data.  This permission will allow the data you authorize below to be accessed for an extended period of time"); ?>
+                            </p>
+                            <p><?php echo xlt("Offline access end date"); ?>: <?php echo $offline_access_date; ?></p>
```

Comment:
text() around the $offline_access_date

Reply:
This is a hard-coded value but I'll change it anyways as someone may pull it from the database.